src/profile-logic/call-tree.js

```diff
@@ -752,7 +756,7 @@ export function computeTracedTiming(
     if (prefixCallNode !== -1) {
       callNodeTotalSummary[prefixCallNode] +=
         callNodeTotalSummary[callNodeIndex];
-      callNodeChildCount[prefixCallNode]++;
+      callNodeHasChildren[prefixCallNode]++;
```

Comment:
do we want `= 1` here? (for consistency mostly, but also to avoid overflow)

Reply:
Oops, yes, that was the intent. Thanks for catching it!